lib/ansible/modules/cloud/amazon/ec2_instance.py

```diff
@@ -141,6 +141,20 @@
       - For T2 series instances, choose whether to allow increased charges to buy CPU credits if the default pool is depleted.
       - Choose I(unlimited) to enable buying additional CPU credits.
     choices: [unlimited, standard]
+  cpu_options:
+    description:
+      - Reduce the number of vCPU exposed to the instance.
+      - Those parameters can only be set at instance launch. The two suboptions threads_per_core and core_count are mandatory.
+      - See U(https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/instance-optimize-cpu.html) for combinations available.
```

Comment:
Probably want a 'requires botocore >= 1.10.16' in this description

Reply:
Added